website_sale_product_description_top/views/website_sale_template.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <template id="product_remove" inherit_id="website_sale_product_description.product">
+        <xpath expr="//p[@id='product_public_description']" position="replace">
+        </xpath>
+    </template>
+    <template id="product" inherit_id="website_sale.product">
+        <xpath
+            expr="//div[@id='product_details']/../div[hasclass('col-md-6', 'col-xl-8')]/t"
```

Comment:
Isn't there a simpler xpath expression to use ? the `hasclass('col-md-6', 'col-xl-8')` could be a bit fragile. Haven't tried but doesn't `//div[@id='o-carousel-product']` work ?

Reply:
Keeping this as it is. If upstream OCA decides to change the xpath expression, I will adjust it here as well.